code/modules/mob/living/stat_states.dm

```diff
@@ -35,9 +35,6 @@
 
 /mob/living/proc/can_be_revived()
 	. = TRUE
-	// if(health <= min_health)
-	if(health <= HEALTH_THRESHOLD_DEAD)
-		return FALSE
```

Comment:
Use the [defib code](https://github.com/ParadiseSS13/Paradise/blob/master/code/game/objects/items/weapons/defib.dm#L459-L464) approach in SR's touch application method instead of doing this

Reply:
I don't think it's a good idea. Please see the comment below (above?) for my reasoning.